content/en/workflows/actions_catalog/_index.md

```diff
@@ -3,11 +3,16 @@ title: Actions Catalog
 kind: documentation
 type: actioncatalog
 disable_toc: true
-further_reading:
-- link: "logs/processing/pipelines"
-  tag: "Documentation"
-  text: "Log processing pipelines"
 cascade:
   disable_sidebar: true
   type: actioncatalog
 ---
+
+<div class="alert alert-warning">Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>
+
+Datadog Workflows provide actions that can be performed against your infrastructure and integrations. You can orchestrate and automate your end-to-end processes by linking together actions for cloud providers, Datadog integrations, Datadog itself, custom HTTP requests, and more. Read [build workflows][1] for more information on creating a workflow.
+
+Click on the buttons below to filter by different parts of Datadog Workflows. A description is available for each action, as well as the action's inputs, outputs, and parameters.
```

Comment:
Removed the part about buttons, since that's a holdover from the security rules page. The action catalog doesn't have buttons.
```suggestion
See below a list of all available workflow actions. Click an action to see its description, inputs, outputs, and parameters.
```